nuget.config

```diff
@@ -1,8 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <packageRestore>
+    <add key="enabled" value="True" />
+    <add key="automatic" value="True" />
+  </packageRestore>
 
-    <packageSources>
-        <add key="Local Packages" value=".\SourceGenerators" />
-    </packageSources>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="Local Packages" value=".\SourceGenerators" />
```

Comment:
Check if the slash '\' should be '/', because the workflow "Continuous Integration / build (pull_request)" failed with:
"/home/runner/work/DomainDrivenDesignUniversity/DomainDrivenDesignUniversity/.\SourceGenerators"